### PR TITLE
Bump oidc lib to version 0.1.5.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: moderator/settings.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'csp.middleware.CSPMiddleware',
+    'mozilla_django_oidc.contrib.auth0.middleware.RefreshIDToken',
```
Comment (johngian) [inline moderator/settings.py]: You also need to add a config entry to store tokens in session.

File: requirements/prod.txt [hunk 2]
```diff
@@ -115,11 +115,17 @@ contextlib2==0.5.4 \
 raven==5.24.3 \
     --hash=sha256:aa1e103e43ad72defa57940e5507e14e2d97c705b5b6ea05a76fa4fc0f85e203 \
     --hash=sha256:0eeaefe0ca72405f934c0abb86d78f6e622d631f55b20c6c67bb293adb059af5
-PyJWT==1.4.2 \
-    --hash=sha256:99fe612dbe5f41e07124d9002c118c14f3ee703574ffa9779fee78135b8b94b6 \
-    --hash=sha256:87a831b7a3bfa8351511961469ed0462a769724d4da48a501cb8c96d1e17f570
-https://github.com/mozilla/mozilla-django-oidc/archive/0.1.2.zip#egg=mozilla_django_oidc \
-    --hash=sha256:021561a7e998ee1f06fd76980690f3bd827f9b65222009079612286ca06667d7
+https://github.com/mozilla/mozilla-django-oidc/archive/0.1.5.zip#egg=mozilla_django_oidc \
```
Comment (johngian) [inline requirements/prod.txt]: Please fix egg definition, it should be ``egg=mozilla-django-oidc==0.1.5``.

**Comment by johngian** [comment]
r+wc

**Pull request merged**